pkgs/applications/networking/irc/quassel/default.nix

```diff
@@ -4,7 +4,7 @@
 , tag ? "-kf5" # tag added to the package name
 , static ? false # link statically
 
-, stdenv, fetchFromGitHub, cmake, makeWrapper, dconf
+, stdenv, fetchFromGitHub, cmake, makeWrapper, dconf, wrapQtAppsHook
```

Comment:
Because the dependency on Qt is unconditional, it is preferred to import `mkDerivation` and use that below instead of `stdenv.mkDerivation`.